s3.rb

```diff
@@ -3,12 +3,39 @@ class S3
 
   def initialize
     @s3 = Aws::S3::Client.new(region: ENV['AWS_REGION'])
+    @keys = ['scientific_name','common_name','redlist_status','iucn_redlist_url']
+    @species = []
   end
 
   def get_file(filename)
     @s3.get_object(*params(filename))
   end
 
+  def get_data(filename)
+    params = {
+      bucket: ENV['AWS_BUCKET'],
+      key: filename,
+      expression_type: 'SQL',
+      expression: "SELECT * FROM S3Object WHERE redlist_status = 'CR' OR redlist_status = 'EN' OR redlist_status = 'VU'",
```

Comment:
Might be better to move the query into it's own method or even a constant if there are no variables involved